hepdata/modules/email/api.py

```diff
@@ -310,6 +311,43 @@ def send_cookie_email(submission_participant,
                            message_body,
                            reply_to_address=coordinator.email)
 
+def send_reminder_email(submission_participant, record_information, version, show_detail=False):
+
+    site_url = current_app.config.get('SITE_URL', 'https://www.hepdata.net')
+    tables = []
+    hepsubmission = get_latest_hepsubmission(
+        publication_recid=record_information['recid']
+    )
+    coordinator = User.query.get(hepsubmission.coordinator)
+
+    reviewers_notified = submission_participant.user_account is not None
+
+    message_body = render_template('hepdata_theme/email/submission_status.html',
+                                   name=submission_participant.full_name,
+                                   actor=coordinator.email,
+                                   article=record_information['recid'],
+                                   message=None,
+                                   invite_token=submission_participant.invitation_cookie,
```

Comment:
The `invite_token` should be set as in `send_notification_email`, i.e.
```python
invite_token = None
if not submission_participant.user_account:
    invite_token = submission_participant.invitation_cookie
```
This ensures that the link to assign permissions is not displayed in `submission_status.html` if the permissions have already been claimed.